### ISSUE TITLE
Support eslint 5

### ISSUE THREAD

**Issue by jingsam** [prettier/prettier-eslint-cli#143]
ESlint 5 have released for a period of time, would you considering add support of it?

- `prettier-eslint-cli` version:
- `prettier` version:
- `eslint` version:

Relevant code/config.

```javascript

```

What you did:

What happened:

Reproduction:

-- paste your link here --

Problem description:

Suggested solution:

**Comment by ryansully** [comment]
There is a deprecation warning that can also be fixed by upgrading to ESLint 5.x.
Details: https://github.com/eslint/eslint/issues/11052

**Comment by marcofugaro** [comment]
Any updates?

**Comment by zimme** [comment]
Tests are fixed in `prettier-eslint` now, so work can begin to support eslint 5+, etc.

**Comment by zimme** [comment]
Fixed in `prettier-eslint-cli@5.0.0` which uses `prettier-eslint@9.0.0`.

**Comment by marcofugaro** [comment]
Good job @zimme! I tried out the new beta version, and this warning pops up:

> { parser: "babylon" } is deprecated; we now treat it as { parser: "babel" }.

The command I'm running is `prettier-eslint --write src/**/*.{js,jsx,ts,tsx}`
